assets/src/edit-story/utils/context.js

```diff
@@ -18,7 +18,7 @@
  * External dependencies
  */
 import { shallowEqual } from 'react-pure-render';
-import { useCallback, useRef } from 'react';
+import React, { useCallback, useRef } from 'react';
```

Comment:
Unnecessary import? If it's for the docblock one could do `@param {import('react').Context}` or `/** @typedef {import('react').Context} context Context */` and then `@param {Context} context Context`

Reply:
Updated.

> Did you try to run any speed tests or profiling on this before/after? I don't really have any suggestions to how though. Just to see if there's a measurable gain here. I like it, but only if the benefits are significant.

See: https://docs.google.com/document/d/18m9HamHkOvvTrv-mSOJawha0K4KTdn49HrnfXShULDI/edit#heading=h.4bti0lhcgu9m